static/docs/index.html

```diff
@@ -8,7 +8,7 @@
     <rapi-doc 
         spec-url = "/oas/v1/openapi.yml"
         show-header	="false"
-        server-url = "http://localhost:9001/api/v1" 
+        server-url = "http://localhost:9002/api/v1" 
```

Comment:
Same, this should be in its own PR with the server.go change, but great catch

Reply:
This was already changed by Dr. Christensen. You need to do a get pull. You have merge conflicts and this is one of them. You are okay to push this.